docs/cookbook/sylius-cloud/introduction-to-sylius-cloud.rst

```diff
@@ -0,0 +1,124 @@
+It's based on its own containers architecture, with gives you all that is good of performance, security and scalability at once.
+
+Using Platform.sh as a base of Sylius Cloud adds a possibility of hosting your Sylius platform on one of the following cloud service providers:
+
+- \- Amazon Web Services (AWS)
+- \- Google Cloud Platform (GCP)
+- \- Microsoft Azure
+- \- OVH
+
+With Sylius Cloud you're able to choose the service provider, server characteristics (number of CPUs, memory amount, disk size etc.)
+with a clear information about costs. It's a good solution both for small, medium and huge web stores with a big traffic and number of orders.
+
+Main Sylius Cloud concepts:
+
+- \- Git-driven architecture - helps with setting up new environments with a simple pushing a new git branch to the specified remote. It makes you ability to create new environments fast.
+- \- Infrastructure as a code - all you need to do is to write your application and a few Yaml files to create your environment. So simple and so powerful.
+- \- Multi-services and multi-apps - it doesn't matter your application is monolith- or microservice-based architecture. It doesn't matter how much different external services you need (MySQL, PostgreSQL, ElasticSearch, Redis etc.) - it just works good in all cases.
+
+Sylius Cloud configuration in Sylius
+------------------------------------
+
+All the infrastructure configuration files are placed in `Sylius-Standard <https://github.com/Sylius/Sylius-Standard>`_ project. It means you'll get it when running the command:
+
+.. code-block:: bash
+
+    composer create-project sylius/sylius-standard acme
+
+The paths you should be interested in are:
+
+- \- `.platform.app.yaml` - This file is used to define the configuration of the application itself. It specifies how the application should be built, what runtime to use, which commands to run on deployment, etc. It also includes information such as the type of application, language runtime, build and deploy hooks, and more.
+- \- `.env.platform.dist` - The environment file automatically used by the application during the build process.
+- \- `.platform/routes.yaml` - This file defines the routes and how traffic is routed to the application. It specifies the incoming URLs and how they should be directed to the appropriate services or applications. Routes can be configured to handle different paths, domains, or other conditions.
+- \- `.platform/services.yaml` - This file is used to define additional services or dependencies required by the application. It allows defining various types of services such as databases, caches, search engines, etc., along with their configuration. Services defined in this file are automatically provisioned and integrated with the application environment.
+
+Creating a new project and environment
+--------------------------------------
```

Comment:
This seems to be doubled with `docs/cookbook/sylius-cloud/basic-project-setup.rst`, but there is more about CI solution, I'm more into putting the all steps, UI or CI within one section if it's related to one thing